compiler+runtime/include/cpp/clojure/string_native.hpp

```diff
@@ -1,5 +1,20 @@
 #pragma once
 
-#include <jank/c_api.h>
+#include <jank/runtime/object.hpp>
 
-extern "C" jank_object_ref jank_load_clojure_string_native();
+namespace clojure::string_native
+{
+  using namespace jank;
+  using namespace jank::runtime;
+
+  object_ref blank(object_ref s);
```

Comment:
When I initially added these, I did so with `const`, which actually caused issues with the cpp interop. Pretty sure we don't need `const` here, so I removed them, but wanted to mention it as it might be the reason we are having issues with cpp interop in other places.